pkg/policy/policy.go

```diff
@@ -42,6 +44,21 @@ type Check struct {
 	Reason       string    `hcl:"reason,optional"`
 }
 
+type Analytic struct {
+	// Whether policy will persist in database
+	Persistence bool
+	// Whether policy was defined in configuration
+	UsedConfig bool
```

Comment:
If we merge https://github.com/cloudquery/cloudquery/pull/841 , I think we can avoid the "UsedConfig" field. It will also simplify the logic of the `Get` function.